samples/node/src/connectedcarsapi.js

```diff
@@ -0,0 +1,143 @@
+const fs = require('fs')
+const util = require('util')
+const readFileAsync = util.promisify(fs.readFile)
+const { JwtUtils } = require('@connectedcars/jwtutils')
+const axios = require('axios')
+
+const _readServiceAccountFile = async fileName => {
+  const fileContent = await readFileAsync(fileName, {
+    encoding: 'utf8'
+  })
+
+  const [ccInfo, rsa] = fileContent.split(
+    '----- END CONNECTEDCARS INFO -----\n'
+  )
+
+  const ccRegex = /----- BEGIN CONNECTEDCARS INFO -----\niss: (.*)\naud: (.*)\nkid: (.*)\n/
+  const [totalMatch, iss, aud, kid] = ccInfo.match(ccRegex)
+
+  if (!(iss && aud && kid)) {
+    throw new Error('Malformed service account file')
+  }
+
+  return {
+    iss,
+    aud,
+    kid,
+    rsa
+  }
+}
+
+const _getServiceToken = async ccServiceAccountFile => {
+  try {
+    const parsedServiceAccountInfo = await _readServiceAccountFile(
```

Comment:
Why are we reading and parsing the file every time, I think it would be better if the parsing was done once in the constructor and the interface would take a string or byte array so the file reading is done by the one using the api.